lib/blas_magma.cu

```diff
@@ -11,7 +11,8 @@
 #define MAX(a, b) (a > b) ? a : b;
 #endif
 
-#define MAGMA_17 //default version version of the MAGMA library
+//#define MAGMA_2X //default version version of the MAGMA library
```

Comment:
I'm thinking we should just remove all support for MAGMA 1.x, and make it MAGMA 2.x only.  Will simplify this file considerably.